forge/src/main/java/org/openjdk/skara/forge/PullRequest.java

```diff
@@ -135,4 +135,5 @@
      * @return
      */
     boolean isDraft();
+    void makeNotDraft();
```

Comment:
While symmetrical, perhaps `makeFinal` or `makeReady` would look nicer.. But if so `isDraft` should perhaps be `isFinal` as well.

Reply:
Yeah, I sort of agree. However both platforms (GitHub and GitLab) does not have a word for the "not draft" state. So to make it very clear I think I prefer the current verbose (and slightly weird) naming to be make the code clearer.